frontend/src/lib/utils.tsx

```diff
@@ -171,20 +171,18 @@ export const selectStyle: Record<string, (base: Partial<CSSProperties>) => Parti
 }
 
 export function debounce(func: (...args: any) => void, wait: number, immediate: boolean, ...args: any): () => void {
-    let timeout: NodeJS.Timeout | undefined
-    return function () {
-        const context = this // eslint-disable-line
-        function later(): void {
+    let timeout: number | undefined
```

Comment:
FYI it seems like we currently don't use this function anywhere, anyways might be a good idea to keep it and add unit tests.

Reply:
Ah, right. I removed it then – we use Kea breakpoint for debouncing anyway, and this util was written a bit oddly imo.